src/WebApp/wwwroot/css/site.css

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
I'm adding this here, although it relates to multiple files. I suggest to adopt the drafts I posted in the PR discussion. To do so, you need to do the following:

1. Add this line:

    ```suggestion
    ﻿html {
        background-color: #f0f0f0;
    ﻿}
    ﻿.container {
    ﻿    max-width: 500px;
    }
    ```

2. Look at https://github.com/luap42/codidact-design-draft. See the files `register.html` and `login.html` for how to create the box effect. Add only contents within `.container` into the views.
3. Try to use `.form-caption` as the only class for the small captions. (requires the update to Co-Design 0.5.0)
4. Implement the code outside `.container` into Shared/_Layouts.cshtml: Add a header with only a brand element with only a site name "Codidact".

Ping me for any questions here or in Discord. :)

Reply:
1. Done
2. Done
3. Done
4. Done

![image](https://user-images.githubusercontent.com/6649900/77777035-4e6ba600-7025-11ea-828b-828198db629a.png)